contracts/registry/src/error.rs

```diff
@@ -26,4 +26,7 @@ pub enum Error {
     AdminOnly = 10,
     /// New version must be greater than the most recent version
     VersionMustBeGreaterThanCurrent = 11,
+    /// Invalid name.
+    /// Must be 64 characters or less and be alphanumeric or '_'
```

Comment:
Looks like there's a couple other rules? Should we say them here?

```suggestion
    /// Must be 64 characters or less and be alphanumeric or '_', and start with a letter
```

Reply:
Can we say `and contain only alphanums & '-'`? Is that a thing, "alphanums"?

Reply:
Technically it's ascii alphanumeric. And I think it's clearer than "alphanums" as I've never heard that and a quick search turned up nothing but a broken site.